R/drug_parameters.R

```diff
@@ -3,37 +3,104 @@
 #' @details Use a vector of preset parameters for the SP-AQ drug (sulphadoxine-pyrimethamine and amodiaquine)
 #' @details Default parameters, from L to R, are: drug_efficacy: 0.9, drug_rel_c: 0.32, drug_prophylaxis_shape: 4.3, drug_prophylaxis_scale: 38.1
 #' @export
-SP_AQ_params <- c(0.9, 0.32, 4.3, 38.1)
+SP_AQ_params <- list("falciparum", c(0.9, 0.32, 4.3, 38.1))
+
+#' @title Preset parameters for the CQ drug for vivax
+#' @description Efficacy from SI of Nekkab et al., DOI: 10.1371/journal.pmed.1003535 (2021),
+#' shape and scale consistent with a longer prophylaxis time: 28 days, decreasing gradually
+#' @details Use a vector of preset parameters for the CQ drug (chloroquine)
+#' @details Default parameters, from L to R, are: drug_efficacy: 0.899, drug_rel_c: 0.5, drug_prophylaxis_shape: 20, drug_prophylaxis_scale: 5
+#' @export
+CQ_params_vivax <- list("vivax", c(0.899, 0.5, 5, 20))
+
+#' @title Preset parameters for the CQ-PQ drug
+#' @description Efficacy from SI of Nekkab et al., DOI: 10.1371/journal.pmed.1003535 (2021),
+#' BS shape and scale consistent with a longer prophylaxis time: 28 days, decreasing gradually,
+#' LS shape are scale consistent with a 7 day PQ treatment, with rapid decay.
+#' @details Use a vector of preset parameters for the CQ-PQ drug (chloroquine and primaquine)
+#' @details Default parameters, from L to R, are: drug_efficacy: 0.948, drug_rel_c: 0.5, drug_prophylaxis_shape: 5, drug_prophylaxis_scale: 20,
+#' drug_hypnozoite_efficacy: 0.713, drug_hypnozoite_prophylaxis_shape: 10, drug_hypnozoite_prophylaxis_scale: 5.5
+#' @export
+CQ_PQ_params_vivax <- list("vivax", c(0.948, 0.5, 5, 20, 0.713, 10, 5.5))
+
+#' @title Preset parameters for the CQ-TQ drug
+#' @description Efficacy from SI of Nekkab et al., DOI: 10.1371/journal.pmed.1003535 (2021),
+#' BS shape and scale consistent with a longer prophylaxis time: 28 days, decreasing gradually,
+#' LS shape are scale consistent with a single TQ treatment, with longer prophylaxis: 45 days.
+#' @details Use a vector of preset parameters for the CQ-TQ drug (chloroquine and tafenoquine)
+#' @details Default parameters, from L to R, are: drug_efficacy: 1, drug_rel_c: 0.5, drug_prophylaxis_shape: 5, drug_prophylaxis_scale: 20,
+#' drug_hypnozoite_efficacy: 0.713, drug_hypnozoite_prophylaxis_shape: 5.5, drug_hypnozoite_prophylaxis_scale: 30
+CQ_TQ_params_vivax <- list("vivax", c(1, 0.5, 5, 20, 0.713, 5, 30))
+
+#' @title Preset parameters for the White et a., ACT drug
+#' @description Efficacy from SI of White et al., DOI: 10.1371/journal.pmed.1003535 (2021),
+#' BS shape and scale consistent with AL prophylaxis in falciparum
+#' @details Use a vector of preset parameters for the ACT drug (Artemisinin combination therapy)
+#' @details Default parameters, from L to R, are: drug_efficacy: 1, drug_rel_c: 0.5, drug_prophylaxis_shape: 11.3, drug_prophylaxis_scale: 10.6
+#' @export
+white_ACT_params_vivax <- list("vivax", c(0.899, 0.5, 11.3, 10.6))
```

Comment:
In general, I'd prefer no to start including author names in objects/parameters as it can proliferate quickly and is not usually immediately useful in the code. 
Could we rename `ACT_params_vivax` and just cite White in the descriptions if needed?

Reply:
I've taken names out (next push).